sia-react-native/screens/Home.js

```diff
@@ -1,4 +1,252 @@
+
+            <View style={{ backgroundColor: "#15202A", paddingTop: 20, paddingBottom: 20, borderBottomWidth: 1, borderBottomColor: '#38444C', paddingHorizontal: 20 }}>
+              <Text style={{ fontSize: 24, color: 'white', fontWeight: '700' }}>
+                School News
+                                </Text>
+              <Text style={{ fontWeight: '300', color: "white", marginTop: 10, marginBottom: 10 }}>
+                Howard University Financial Aid Scandal Continues
+
+                                </Text>
+
+              <View style={{ width: width - 40, height: 300, marginTop: 10 }}>
+                <Image
+                  style={{ flex: 1, height: null, width: null, resizeMode: 'cover', borderRadius: 10, borderWidth: 1, borderColor: '#38444C' }}
+                  source={require('../assets/tyrone.png')}
+                />
+
+              </View>
+
+
+              <View style={{ marginTop: 0, paddingHorizontal: 2 }}>
+                <Text style={{ fontWeight: '100', color: 'white', marginTop: 20 }}>
+                  View all news articles
+
+                                  </Text>
+              </View>
+            </View>
+
+            <View style={{ backgroundColor: "#15202A", paddingTop: 20, marginBottom: 40 }}>
+              <Text style={{ fontSize: 24, color: 'white', fontWeight: '700', paddingHorizontal: 20 }}>
+                Guide Links
+                            </Text>
+              <View style={{ paddingHorizontal: 20, marginTop: 20, flexDirection: 'row', flexWrap: 'wrap', justifyContent: 'space-between' }}>
+                <Links width={width}
+                  name="Handshake"
+                />
+                <Links width={width}
+                  name="Campus Directory"
+                />
+                <Links width={width}
+                  name="Athletics"
+                />
+
+                <Links width={width}
+                  name="Shuttle"
+                />
+
+                <Links width={width}
+                  name="Virtual Map"
+                />
+
+                <Links width={width}
+                  name="ETS"
+                />
+                <View style={{ marginTop: 0, paddingHorizontal: 2 }}>
+                  <Text style={{ fontWeight: '100', color: 'white', marginTop: 10, marginBottom: 30 }}>
+                    View all links
+
+                                  </Text>
+                </View>
+
+
+              </View>
+            </View>
+          </ScrollView>
+
+        </View>
+      </SafeAreaView>
+    );
+  }
+}
+export default Home;
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    alignItems: 'center',
+    justifyContent: 'center',
+    backgroundColor: "#eaeaea"
+  },
+  shadow: {
+    shadowColor: "#000",
+    shadowOffset: {
+      width: 0,
+      height: 5,
+    },
+    shadowOpacity: 0.34,
+    shadowRadius: 6.27,
+
+    elevation: 10,
+  }
+});
+
+
+
+/*import React from 'react'
 import {
```

Comment:
Group imports at the top of the file.

Reply:
Again didn't want to mix up imports because it would get confusing to me